product_service_duration/models/product_template.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+import odoo.addons.decimal_precision as dp
+
+
+class ProductTemplate(models.Model):
+
+    _inherit = 'product.template'
+
+    service_user_id = fields.Many2one(
```

Comment:
This actually needs to be to a resource, not to a user. The resource is the user in the context of their working hours.